__tests__/Unit/Components/Tasks/TaskDetails.test.tsx

```diff
@@ -0,0 +1,53 @@
+import { render, screen } from '@testing-library/react';
+import TaskDetails from '../../../../src/components/taskDetails/index';
+
+const details = {
+  url: 'https://realdevsquad.com/tasks/6KhcLU3yr45dzjQIVm0J/details',
+  taskID: '6KhcLU3yr45dzjQIVm0J',
+  isNoteworthy: true,
+  lossRate: {
+    dinero: 0,
+    neelam: 0,
+  },
+  purpose: 'string',
+  endsOn: 1618790400,
+  title: 'test 1 for drag and drop',
+  status: 'assigned',
+  assignee: 'ankur',
+  links: ['null'],
+  dependsOn: ['null'],
+  percentCompleted: 0,
+  type: 'feature',
+  startedOn: 1618790410,
+  featureUrl: 'string',
+  completionAward: {
+    neelam: 0,
+    dinero: 110,
+  },
+};
```

Comment:
isnt this data present in task details handler why are we re-writing it?

is it required here? if yes, cant we use common constants for both files

Reply:
Yes, It is already present in the task-details handler. But I wanted to use this Object's property values as expected values, once I get the mocked response data, but as I am no more considering the mocked data, I will remove these all properties except `url `and `taskID `which I will be passing as props.